### ISSUE TITLE
Split out "alumni" from Organisation's "employees and affiliated" 

### ISSUE THREAD

**Issue by egonw** [fnielsen/scholia#739]
Currently, the "employees and affiliated" section on the Organization aspect also shows alumni, e.g. Cristian here:

![image](https://user-images.githubusercontent.com/26721/59340090-9ffcab80-8d05-11e9-9d89-6ca28bae9c04.png)

The info is easily encoded (tho often missing) in Wikidata (ShEx anyone?):

![image](https://user-images.githubusercontent.com/26721/59340144-b6a30280-8d05-11e9-945a-81eda03434fc.png)

I propose a "Alumni" section on the organisation page. If no "end date" is given, it's an active employee.

Comments please.

**Comment by Daniel-Mietchen** [comment]
Yes, splitting gout alumni makes sense, at least aspirationally and in the long run. It is pragmatic to treat those with no end date as active employees (although the often missing information messes this up quite a bit), but the reverse is not necessarily the case, as some people come to a given institution for several periods.

In the short run, perhaps we should start by adding some queries about missing affiliation/ employment data (and dates) to the /missing page for organizations.
